setup.py

```diff
@@ -1,11 +1,24 @@
+#!/usr/bin/env python
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
+import re
+
 try:
-    from setuptools import setup
+    import setuptools as setuplib
```

Comment:
This try/catch allows someone to install this locally if they don't have `setuptools` installed, e.g. using `python setup.py install`. That means that they can't get any of the extras (that's a `setuptools` feature) but they can at least install the package.